apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -961,8 +964,10 @@ const Ticket = new GQLListSchema('Ticket', {
                 ]
             )(...args)
 
-            /* NOTE: this sends different kinds of notifications on ticket create/update */
-            await sendTicketChangedNotifications.delay({ ticketId: updatedItem.id, existingItem, operation })
+            /* NOTE: this sends different kinds of notifications on ticket create/update except bulk update operation */
+            if (!DISABLE_PUSH_NOTIFICATION_FOR_OPERATIONS.includes(get(context, ['req', 'body', 'operationName'], null))) {
```

Comment:
It is impossible to assume that the logic of notifications will be tied to the name of the GQL mutation. This solution seems non-obvious and requires separate discussion.